benchmark/src/test/scala/com/evolutiongaming/kafka/journal/replicator/BatchBenchmark.scala

```diff
@@ -0,0 +1,364 @@
+package com.evolutiongaming.kafka.journal.replicator
+
+import cats.data.NonEmptyList as Nel
+import cats.syntax.all.*
+import com.evolutiongaming.kafka.journal.*
+import com.evolutiongaming.skafka.Offset
+import scodec.bits.ByteVector
+import org.openjdk.jmh.annotations.{Benchmark, Fork, Measurement, Scope, State, Warmup}
+import org.openjdk.jmh.infra.Blackhole
+
+import java.time.Instant
+import java.util.concurrent.TimeUnit
+
+/** To run benchmarks: {{{sbt benchmark/Jmh/run com.evolutiongaming.kafka.journal.replicator.BatchBenchmark}}}
+ */
+@State(Scope.Benchmark)
+@Fork(1)
+@Warmup(iterations = 1, time = 10, timeUnit = TimeUnit.SECONDS)
+@Measurement(iterations = 5, time = 10, timeUnit = TimeUnit.SECONDS)
+class BatchBenchmark {
+
+  @Benchmark
+  def original(blackhole: Blackhole): Unit = {
+    val of = Batch_4_1_0.of
+    callAll[Batch_4_1_0](blackhole, of)
+  }
+
+//  @Benchmark
```

Comment:
Please either remove commented out code or leave a code comment above it on what's its purpose and why it should stay in the source code. Just to keep the codebase tidy.